%YAML:1.0
---
#image_width: 1280
#image_height: 720

camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [576.03765948,0.,311.15853278,
           0.,578.23132687,242.0768764,
           0.,0.,1.]
distortion_coefficients: !!opencv-matrix
   rows: 4
   cols: 1
   dt: d
   data: [ -0.25281989, 0.09467408,
       -0.00041095, 0.00024042]
